setup.py

```diff
@@ -60,4 +60,8 @@
         'scipy == 0.17.1',
     ],
 
+    # pytest integration
+    setup_requires=['pytest-runner'],
```

Comment:
Thought about this a bit more. @dhimmel I changed my mind, we __should__ specify versions to ensure our build is stable while allowing for manual upgrades where necessary. @vasudevanv I can take a look and make sure the dependencies are correctly specified in setup.py, but this looks ok to me. I will do an independent pull request review this afternoon as well.